pkg/kubeconfig/kubeconfig.go

```diff
@@ -69,11 +69,13 @@ func BuildKubernetesClientset(s *state.State) error {
 		return err
 	}
 
-	s.DynamicClient, err = client.New(s.RESTConfig, client.Options{})
+	dynamicClient, err := client.New(s.RESTConfig, client.Options{})
```

Comment:
This doesn't change anything, as if client.New() returns error it also returns nil client, meaning we assign nil to already nil (no actual change).

Reply:
As described above, client.New() returns a nil interface. This looks something like:

`*client.client(nil)` instead of real `nil` and does not evaluate == nil as true